### PR TITLE
Bernard

### PR DESCRIPTION
**Pull request by fsansalvadore**

### CODE REVIEW

File: apps/www/components/LaunchWeek/X/Releases/data/lwx_data.tsx [hunk 1]
```diff
@@ -237,19 +236,44 @@ const days: WeekDayProps[] = [
...
+    shipped: true,
+    isToday: true,
+    hasCountdown: false,
+    blog: '/blog/introducing-read-replicas',
     date: '15 Dec',
     published_at: '2023-12-15T08:00:00.000-08:00',
-    description: '',
+    description: (
+      <>
+        Introducing <strong>Read Replicas</strong> for High Availability
```
Comment (fsansalvadore) [inline apps/www/components/LaunchWeek/X/Releases/data/lwx_data.tsx:247]: @kiwicopple thoughts on this announcement title?

File: apps/www/_blog/2023-12-15-introducing-read-replicas.mdx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+### Geo-based load balancing
+
+Currently, we have implemented a round-robin strategy of routing PostgREST requests to the different databases. This is useful for sharing the load across the primary and the read replica, but in cases where latency is important, you might want to always route to the closest database instead.
+
+### Supabase Products leveraging Replicas
+
+We are working on support for database-level load balancing through [Supavisor](https://supabase.com/blog/supabase-branching). When routing database traffic through Supavisor, it routes write queries to the primary and splits read queries between the primary and read replica. This distributes load to all your databases without any changes to your application. Other Supabase products like Auth, Storage, and Realtime use the database heavily. We are working on adding read replica support to these products, so that they can leverage the read replica when appropriate. This means they won’t just query the primary but will also use the read replica’s compute when possible.
```
Comment (RamiroND) [inline apps/www/_blog/2023-12-15-introducing-read-replicas.mdx:131]: Here we are linking to branching post, should it be to the supavisor post?

Comment (dragarcia) [reply apps/www/_blog/2023-12-15-introducing-read-replicas.mdx:131]: updating

Comment (dragarcia) [reply apps/www/_blog/2023-12-15-introducing-read-replicas.mdx]: updated

**Review by RamiroND — APPROVED** [review]

**Review by dragarcia — APPROVED** [review]

**Pull request merged**